feature-libs/checkout/components/components/delivery-mode/delivery-mode.component.html

```diff
@@ -1,128 +1,65 @@
-<ng-container *cxFeatureLevel="'!4.2.2'">
-  <div [formGroup]="mode">
-    <div class="row">
-      <div class="col-md-12 col-lg-9">
-        <h2 class="cx-checkout-title d-none d-lg-block d-xl-block">
```

Comment:
Can you update the breaking changes list in `docs/migration/5_0.md`, and remove some entries which are no longer applicable?

Reply:
Hi Nik,

Thanks a lot for your reminder! Yes definitely I will update 5_0.md once all changes are done and well tested.

Reply:
Hi Nik,

Thanks a lot for your reminder! Yes definitely I will update 5_0.md once all changes are done and well tested.